ducktape/cluster/remoteaccount.py

```diff
@@ -313,28 +298,32 @@ def ssh_capture(self, cmd, allow_fail=False, callback=None, combine_stderr=True,
         stdout = chan.makefile('r', -1)
         stderr = chan.makefile_stderr('r', -1)
 
+        exit_status = [None]
+
         def output_generator():
 
             for line in iter(stdout.readline, ''):
+                self.logger.debug(line)
```

Comment:
This is likely to be excessive to log for some use cases. e.g., because the test driver needs to analyze the output, verifiable_producer/consumer stream directly to the driver node: https://github.com/apache/kafka/blob/trunk/tests/kafkatest/services/verifiable_consumer.py#L238. Depending on how fast events are being generated, that could turn into a massive log (and for some tests that try to stress the system and expose potential concurrency issues, we could be generating events quite quickly).

Reply:
This makes sense. I added an overridable limit of 5MB. Testing now